platform/mikro-e/Readme.md

```diff
@@ -34,16 +34,16 @@ environment variables and compile an example, say "Hello World" which is include
 
     $ export PATH=$PATH:/opt/microchip/xc32/v1.34/bin/
 
-Platform mikro-e makefile has been updated to build either for cc2520 or for ca8210 depending the build option USE_CC2520 or USE_CA8210. 6lowpan channel(default 26) and pan_id(default 0xabcd) can be passed from makefile options. For debugging purposes, UART3 will be used by default, but USE_SERIAL_PADS can be selected for UART2.The application e.g. Hello World can be built for Mikro-E Clicker and HEX file can be generated to flash on the board along with possible build options as per below:
+Platform mikro-e makefile has been updated to build either for cc2520 or for ca8210 depending the build option USE_CC2520 or USE_CA8210 (USE_CA8210 is set to 1 by default). 6lowpan channel(default 26) and pan_id(default 0xabcd) can be passed from makefile options. For debugging purposes, UART3 will be used by default, but USE_SERIAL_PADS can be selected for UART2.The application e.g. Hello World can be built for Mikro-E Clicker and HEX file can be generated to flash on the board along with possible build options as per below:
 
 For clicker boards using CA8210:
 
     $ cd examples/hello-world
-    $ make TARGET=mikro-e USE_CA8210=1 CHANNEL=26 PAN_ID=0xabcd USE_SERIAL_PADS=1
+    $ make TARGET=mikro-e CHANNEL=26 PAN_ID=0xabcd USE_SERIAL_PADS=1
     $ /opt/microchip/xc32/v1.34/bin/xc32-bin2hex hello-world.mikro-e
 
 For clicker boards using CC2520:
 
     $ cd examples/hello-world
-    $ make TARGET=mikro-e USE_CC2520=1 CHANNEL=26 PAN_ID=0abcd USE_SERIAL_PADS=1
+    $ make TARGET=mikro-e USE_CC2520=1 USE_CA8210=0 CHANNEL=26 PAN_ID=0abcd USE_SERIAL_PADS=1
```

Comment:
Now this would be confusing. I think we can get rid of CA8210 macro since its default and can have just USE_CC2520 to force if at all anyone wants to use it with old boards.